<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 1. Соколов Егор, гр. 143, вариант 18.</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Параметры расчета</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButtonDefault">
         <property name="text">
          <string>По умолчанию</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButtonCustom">
         <property name="text">
          <string>Задать вручную</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxA">
           <property name="specialValueText">
            <string/>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>100000000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelA">
           <property name="text">
            <string>a = </string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelMiu">
           <property name="text">
            <string>miu = </string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelM">
           <property name="text">
            <string>m = </string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelY0">
           <property name="text">
            <string>Y_0 =</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxM">
           <property name="specialValueText">
            <string/>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>1000000000000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxY0">
           <property name="specialValueText">
            <string/>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>100000000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBoxMiu">
           <property name="specialValueText">
            <string/>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>100000000000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelCountIntervals">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Количество интервалов разбиения</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxCountIntervals">
         <property name="specialValueText">
          <string/>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000000000.000000000000000</double>
         </property>
         <property name="value">
          <double>16.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCountValues">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Количество выборок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxCountValues">
         <property name="specialValueText">
          <string/>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000000000000.000000000000000</double>
         </property>
         <property name="value">
          <double>1500.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Рассчитать</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Результаты расчета</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCustomPlot" name="histogram" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Матожидание</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditM">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Дисперсия (смещенная)</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditD">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDCorrected">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>99999</height>
            </size>
           </property>
           <property name="text">
            <string>Дисперсия (исправленная)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditDCorrected">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Второй начальный момент</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditMoment2">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Третий начальный момент</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditMoment3">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonClear">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionCalculate"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionCalculate">
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
